### PR TITLE
Routing: add explanation for the "_fragment" parameter

### PR DESCRIPTION
**Pull request by alexisLefebvre**
Add explanation about this PR: https://github.com/symfony/symfony/pull/12979

Presented in http://symfony.com/blog/new-in-symfony-3-2-routing-improvements

I copy-pasted the start of the announcement as the explanation since I couldn't have found a better explanation. I hope that won't be a problem.

### CODE REVIEW

**Comment by xabbuh** [comment]
This change looks good to me. But I think we should add a `versionadded` directive which explains that support for this parameter was introduced in 3.2.

**Comment by alexislefebvre** [comment]
Thanks for the review @xabbuh, I added `versionadded`.

File: routing.rst [hunk 1]
```diff
@@ -475,6 +475,13 @@ that are special: each adds a unique piece of functionality inside your applicat
 ``_format``
     Used to set the request format (:ref:`read more <routing-format-param>`).
 
+``_fragment``
+
+ .. versionadded:: 3.2
```
Comment (javiereguiluz) [inline routing.rst]: I know it's a bit verbose, but in Symfony Docs the `versionadded` directive is used like this:

```
.. versionadded:: 3.2
    The ``_fragment`` parameter was introduced in Symfony 3.2.
```

**Comment by javiereguiluz** [comment]
👍 

Status: reviewed

File: routing.rst [hunk 2]
```diff
@@ -475,6 +475,13 @@ that are special: each adds a unique piece of functionality inside your applicat
 ``_format``
     Used to set the request format (:ref:`read more <routing-format-param>`).
 
+``_fragment``
```
Comment (stof) [inline routing.rst]: IMO, this is the wrong place for it, as you won't use ``_fragment`` in route patterns.

It should rather be explained in a section near ``Generating URLs with Query Strings``

**Comment by alexislefebvre** [comment]
@javiereguiluz I updated the `versionadded` part. What do you think of the comment of stof? Should I add a new section near `Generating URLs with Query Strings`? Thanks.

**Comment by wouterj** [comment]
Hi @alexislefebvre!

I've merged your pull request and fixed a syntax error in 0a22dda. I think it's a nice idea to also document this in the generating section indeed. Not sure if I would add a new section or just add it to the main section.

Feel free to propose another PR doing this. For now, thanks a lot for documenting this nice feature!

**Pull request closed without merging**